expression/errors.go

```diff
@@ -82,3 +82,18 @@ func handleDivisionByZeroError(ctx sessionctx.Context) error {
 	sc.AppendWarning(ErrDivisionByZero)
 	return nil
 }
+
+// unsupportedJSONComparison reports warnings while there is a JSON type in least/greatest function's arguments
+func unsupportedJSONComparison(ctx sessionctx.Context, args []Expression) {
```

Comment:
move this function to builtin_compare.go

Reply:
Done.